packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.tsx

```diff
@@ -133,7 +131,7 @@ export class DatePicker extends BaseComponent<IDatePickerProps, IDatePickerState
     let { formatDate, value } = props;
 
     this.state = {
-      selectedDate: value || new Date(),
+      selectedDate: value || undefined,
```

Comment:
Can't this change break some consumers that would expect the default to be new Date()? Seems like this should be controlled by a prop.